devops/replica_cluster.yaml

```diff
@@ -1986,6 +1993,24 @@ Resources:
                 ulimit -n 20000000
 
                 SEP=$(echo "${KafkaHostname}" | grep -q "?" && echo "&" || echo "?")
+
+                sudo -Eu geth /usr/bin/geth --cache=$allocatesafe --light.maxpeers 0 --maxpeers 0 --rpc --rpcaddr 0.0.0.0 --rpcport 8545 --datadir=/var/lib/ethereum ${FreezerFlags} &
```

Comment:
I think we need ${ReplicaFlags} here to make sure everything is configured for the right chain.